draft-ietf-core-new-block.xml

```diff
@@ -412,20 +406,8 @@
       <section title="Structure of Q-Block1 and Q-Block2 Options">
         <t>The structure of Q-Block1 and Q-Block2 Options follows the
         structure defined in Section 2.2 of <xref
-        target="RFC7959"></xref>.</t>
-
-        <t>There is no default value for the Q-Block1 and Q-Block2 Options.
-        Absence of one of these options is equivalent to an option value of 0
-        with respect to the value of block number (NUM) and more bit (M) that
-        could be given in the option, i.e., it indicates that the current
-        block is the first and only block of the transfer (block number is set
-        to 0, M is unset). However, in contrast to the explicit value 0, which
-        would indicate a size of the block (SZX) of 0, and thus a size value
-        of 16 bytes, there is no specific explicit size implied by the absence
-        of the option -- the size is left unspecified. (As for any uint, the
-        explicit value 0 is efficiently indicated by a zero-length option;
-        this, therefore, is different in semantics from the absence of the
-        option).</t>
+        target="RFC7959"></xref> with Q-Block1 substituted for Block1 and
```

Comment:
I suggest to maintain the old wording of this section and limit the chnages to those required to address the WGLC comments. 

As a reminder, we included some tetx from 7959 on purpose. This is why we have the following: 

"Some text from [RFC7959] is reused for readers convenience."

Reply:
Text reverted

Reply:
Thanks